framework/mesh/MeshGenerator/FromFileMeshGenerator.cc

```diff
@@ -20,7 +20,7 @@ chi::InputParameters FromFileMeshGenerator::GetInputParameters()
 
   params.SetGeneralDescription("Generator for loading an unpartitioned mesh"
                                " from a file.");
-  params.SetDocGroup("MeshGenerator");
+  params.SetDocGroup("doc_MeshGenerators");
```

Comment:
So these are tags to documentation pages and is not user facing. In doxygen you create a page like this
```
\** \defgroup <tag_name> Multiword description
\ingroup <another_tag>

Page contents bla bla 
*\
```

To place this object as a child of a group we use that `SetDocGroup` method otherwise the page for its documentation will go in the main category `Input Reference`. So there is now better way to keep it short and sweet but to just use tags.